### DISCUSSION TITLE
Android parameters /loadbalanceinfo

### DISCUSSION THREAD

**Discussion by Sjoerdvdn** [FreeRDP/FreeRDP#8979] [Q&A]
Hi,

How can I add the /Loadbalanceinfo parameter to a RDP connection.
I need to add below parameters as we connect to a RDS Connection Broker:
loadbalanceinfo:s:tsv://MS Terminal Services Plugin.1.<collection>

**Comment by akallabeth** [comment]
Currently not implemented.
Android client is in maintenance mode until we have more time to bring it up to speed.

`LibFreeRDP.setConnectionInfo` is the place where the command line is generated, but there need to be settings added for the bookmarks as well.

Any help appreciated.
